earthfile2llb/earthfile2llb.go

```diff
@@ -220,6 +240,14 @@ func Earthfile2LLB(ctx context.Context, target domain.Target, opt ConvertOpt, in
 	if err != nil {
 		return nil, err
 	}
+	if opt.MainTargetDetailsFuture != nil {
+		opt.MainTargetDetailsFuture <- TargetDetails{
+			ID:                 sts.ID,
+			EarthlyOrgName:     bc.EarthlyOrgName,
+			EarthlyProjectName: bc.EarthlyProjectName,
+		}
+		opt.MainTargetDetailsFuture = nil
```

Comment:
Should we close the channel here?
```suggestion
        close(opt.MainTargetDetailsFuture)
		opt.MainTargetDetailsFuture = nil
```

Reply:
Nope - not needed.